build_container/build_container_ubuntu.sh

```diff
@@ -18,6 +18,12 @@ update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 1000
 update-alternatives --config gcc
 update-alternatives --config g++
 
+# Google Cloud SDK
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" \
+  | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
```

Comment:
It might be nice to pin here, WDYT?

Reply:
do you mean pinning gcloud to a specific version? it is possible but I don't see much value to do it for now. We can always revisit that later if we see some issue for this picking up specific version of gcloud.

Reply:
No, I mean SHA256 of the GPG key.

Reply:
I believe the gpg key is periodically updated so it is not a good idea to pin either. This is from https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu and it mentions about expired key.